test/e2e/reconciler/locationworkspace/local_apiexport_test.go

```diff
@@ -70,7 +70,7 @@ func TestSyncTargetLocalExport(t *testing.T) {
 		framework.WithExtraResources("services"),
 		framework.WithSyncTargetName(syncTargetName),
 		framework.WithSyncedUserWorkspaces(computeWorkspace),
-	).Start(t)
+	).BootstrapSyncTarget(t)
```

Comment:
Why not `StartSyncer`? Why this cryptic name?

Reply:
It doesn't start the syncer anymore.
The `Start()` function still starts the Syncer as before.

The new, alternate `BootstrapSyncTarget()` method mimics all the actions the Syncer would have done to make the Synctarget usable:
- heartbeat
- API importing
- API authorization checking of APIs in the downstream cluster.

Since I'm quite bad at naming, so I'm very open to another name :-)